css/app.css

```diff
@@ -114,11 +114,11 @@ img {
 }
 
 #tabs-list li {
-  -webkit-box-orient: horizontal;
-  -webkit-box-pack: justify;
-  display: -webkit-flex;
+  display: flex;
+  flex-direction: row;
   align-items: center;
-  cursor: pointer;
+  height: 42px;
+  padding: 0px 0px 0px 20px;
```

Comment:
Is this just `padding-right`? Do you want to specify every padding?

Also if you want to specify each padding, just do

`padding: 0 0 0 20px;`

Reply:
No cause this overrules the padding defined in #sidebar li

Reply:
Sorry 'no' as in 'yes i do want to specify every padding'. Will make the px redundancy update :)

Reply:
Done.